includes/amp-helper-functions.php

```diff
@@ -1590,7 +1600,8 @@ function amp_get_content_sanitizers( $post = null ) {
 		AMP_Accessibility_Sanitizer::class         => [],
 		// Note: This validating sanitizer must come at the end to clean up any remaining issues the other sanitizers didn't catch.
 		AMP_Tag_And_Attribute_Sanitizer::class     => [
-			'prefer_bento' => amp_is_bento_enabled(),
+			'prefer_bento'                  => amp_is_bento_enabled(),
+			'allow_localhost_http_protocol' => amp_is_dev_mode(),
```

Comment:
Minor point, but since we check `amp_is_dev_mode()` below as well, might as well save the value to a variable and use it in both places.

Reply:
Make sense, since there is no caching in `amp_is_dev_mode()`.